app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -33,68 +33,68 @@ keywords:
+Some teacher training courses in England are salaried – this means you [work and earn a salary while you train](/funding-and-support/salaried-teacher-training). Currently, places on these courses are very limited. You will need a Skilled Worker visa if you train on a salaried training course. Your Skilled Worker visa must be sponsored by the school employing you.
 
-### To apply for your visa, follow these steps 
+### To apply for your visa, follow these steps
 
-Visit [Find postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses) and filter your search results by visa sponsorship. 
+Visit [Find postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses) and filter your search results by visa sponsorship.
 
-Email your chosen provider to check they can sponsor you, then [apply for your teacher training course](/how-to-apply-for-teacher-training). 
+Email your chosen provider to check they can sponsor you, then [apply for your teacher training course](/how-to-apply-for-teacher-training).
 
-When you have a confirmed offer of a training place, you’ll be able to apply for your visa.  
+When you have a confirmed offer of a training place, you’ll be able to apply for your visa.
 
-For a [Student visa](https://www.gov.uk/student-visa), you will also need to: 
+For a [Student visa](https://www.gov.uk/student-visa), you will also need to:
 
 * have a reference number, called a confirmation of acceptance for studies (CAS), which your training provider will give you
 * [have enough money to support yourself and pay for your course](https://www.gov.uk/student-visa/money)
 * [meet certain standards in speaking, reading, writing and understanding English](https://www.gov.uk/student-visa/knowledge-of-english)
 
-For a [Skilled Worker visa](https://www.gov.uk/skilled-worker-visa), you will also need to: 
+For a [Skilled Worker visa](https://www.gov.uk/skilled-worker-visa), you will also need to:
 
-* have a ‘certificate of sponsorship’ from the school which is paying you, confirming they will sponsor you 
+* have a ‘certificate of sponsorship’ from the school which is paying you, confirming they will sponsor you
 * [meet certain standards in speaking, reading, writing and understanding English](https://www.gov.uk/skilled-worker-visa/knowledge-of-english)
-* check your salary meets the requirement for a Skilled worker visa 
+* check your salary meets the requirement for a Skilled worker visa
 
-### Salary requirement for a Skilled Worker visa 
+### Salary requirement for a Skilled Worker visa
 
-Non-UK salaried trainee teachers must earn a minimum salary to be eligible for a Skilled Worker visa – this varies by region, as follows: 
+Non-UK salaried trainee teachers must earn a minimum salary to be eligible for a Skilled Worker visa – this varies by region, as follows:
 
-* inner London: £25,831 
+* inner London: $salaries_skilledworkervisatraineeminfulltime_innerlondon$
```

Comment:
This is hard to read, suggest salaries_skilledworkervisatrainee_fulltime_innerlondonmin

Reply:
resolved in above comment